### DISCUSSION TITLE
ESP32-S3-EYE MicroPython Bootloader Problem

### DISCUSSION THREAD

**Discussion by mmurat06** [micropython/micropython#10121] [Using MicroPython]
I downloaded and uploaded the  micropython to ESP32-S3 (.bin file from "https://micropython.org/download/GENERIC_S3/"). When I try to upload any code to my board(**ESP32-S3_EYE**)  I am facing with this error:

---****----
PROBLEM IN THONNY'S BACK-END: Internal error (serial.serialutil.SerialTimeoutException: Write timeout).
See Thonny's backend.log for more info.
You may need to press "Stop/Restart" or hard-reset your MicroPython device and try again.
---****----

**Are there anyone can use micropython on this board? Could you please help me?**

**Comment by jimmo** [comment]
@mmurat06 Have you tried using esptool.py directly, as per the instructions at https://micropython.org/download/GENERIC_S3/ ?

**Comment by Shek20** [comment]
@mmurat06 Did you find a solution.

**Comment by scruss** [comment]
The ESP32-S3-EYE has “… [8 MB Octal PSRAM](https://www.espressif.com/en/products/devkits/esp-eye/overview.)”, so you'd have more luck using one of the [Firmware (Support for Octal-SPIRAM)](https://micropython.org/download/ESP32_GENERIC_S3/) downloads.

Also seconding Jimmo's suggesting that you try using esptool directly. Thonny doesn't show you all the results you need to diagnose problems.
